_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw_utils_new.cpp

```diff
@@ -2516,19 +2535,28 @@ mfxStatus MfxHwH264Encode::CodeAsSkipFrame(     VideoCORE &            core,
     else
     {
         mfxI32 fid = task.m_fid[0];
-        MFX_CHECK(task.m_list0[fid].Size(), MFX_ERR_UNDEFINED_BEHAVIOR);
-        DpbFrame& refFrame = task.m_dpb[0][task.m_list0[fid][0] & 127];
+        mfxU8 ind = ((task.GetFrameType() & MFX_FRAMETYPE_B)  && task.m_list1[fid].Size() != 0) ? task.m_list1[fid][0] : task.m_list0[fid][0];
+        MFX_CHECK(ind < 15, MFX_ERR_UNDEFINED_BEHAVIOR);
```

Comment:
why 15? Can this be used as some macro or const int with the reasonable name instead of magic number?